app/models/rent.rb

```diff
@@ -107,10 +107,10 @@ def update_without_validation(params)
     st = self.status
     self.attributes = params;
     self.save(validation: false)
-    if(ak == true && !self.aktiv)
+    if (ak == true && !self.aktiv)
```

Comment:
Don't use parentheses around the condition of an `if`.<br>Redundant `self` detected.